packages/firebase/functions/src/event/event.ts

```diff
@@ -196,84 +209,110 @@ export const eventData: Record<string, IEventData> = {
     eventObject: async (proposalId: string): Promise<any> => (await proposalDb.getProposal(proposalId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    notifyUserFilter: async (proposal: any): Promise<string[]> => {
+
+    notifyUserFilter: async (proposal: IProposalEntity): Promise<string[]> => {
       return [proposal.proposerId];
     }
   },
   [EVENT_TYPES.PAYMENT_FAILED]: {
     eventObject: async (proposalId: string): Promise<any> => (await proposalDb.getProposal(proposalId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    notifyUserFilter: async (proposal: any): Promise<string[]> => {
+
+    notifyUserFilter: async (proposal: IProposalEntity): Promise<string[]> => {
       return [proposal.proposerId];
     }
   },
   [EVENT_TYPES.SUBSCRIPTION_CANCELED_BY_PAYMENT_FAILURE]: {
     eventObject: async (subscriptionId: string): Promise<any> => (await subscriptionDb.get(subscriptionId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    notifyUserFilter: async (subscription: any): Promise<string[]> => {
+
+    notifyUserFilter: async (subscription: ISubscriptionEntity): Promise<string[]> => {
       return [subscription.userId];
     }
   },
   [EVENT_TYPES.SUBSCRIPTION_PAYMENT_CONFIRMED]: {
     eventObject: async (paymentId: string): Promise<any> => (await paymentDb.get(paymentId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+
     notifyUserFilter: async (payment: IPaymentEntity): Promise<string[]> => {
       return [payment.userId];
     }
   },
   [EVENT_TYPES.SUBSCRIPTION_CANCELED_BY_USER]: {
     eventObject: async (subscriptionId: string): Promise<any> => (await subscriptionDb.get(subscriptionId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    notifyUserFilter: async (subscription: any): Promise<string[]> => {
+
+    notifyUserFilter: async (subscription: ISubscriptionEntity): Promise<string[]> => {
       return [subscription.userId];
     }
   },
   [EVENT_TYPES.REQUEST_TO_JOIN_EXECUTED]: {
     eventObject: async (proposalId: string): Promise<any> => (await proposalDb.getProposal(proposalId)),
-    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-    notifyUserFilter: async (proposal: any): Promise<string[]> => {
+
+    notifyUserFilter: async (proposal: IProposalEntity): Promise<string[]> => {
       return [proposal.proposerId];
     }
   },
+  [EVENT_TYPES.DISCUSSION_MESSAGE_REPORTED]: {
+    eventObject: async (discussionMessageId: string): Promise<any> => (await getDiscussionMessageById(discussionMessageId)).data(),
+
+    notifyUserFilter: async (discussionMessage: IDiscussionMessage): Promise<string[]> => {
+      const common = await commonDb.get(discussionMessage.commonId)
+      return [common.members[0].userId]; // send to founder for now, need to send to all moderators
```

Comment:
Not like this. There is no guarantee that the members will be always like this (they should be, but you never know and there are no protections in place), so maybe: 

```suggestion
      return [common.metadata.founderId]; // send to founder for now, need to send to all moderators
```

What do you think?

Reply:
Yes, I agree :)